events-api.go

```diff
@@ -0,0 +1,107 @@
+package kable
+
+import (
+	"context"
+	"encoding/json"
+	"github.com/Kable-io/kable-go/internal/openapi"
+	"github.com/ehsaniara/gointerlock"
+	"io"
+	"log"
+	"time"
+)
+
+type (
+	Event openapi.Event
+)
+
+type EventsApi struct {
+	api     *openapi.APIClient
+	options *KableOptions
+	ctx     context.Context
```

Comment:
what is the context here? where does that come from? is that a Go concept I don't know?

Reply:
Context provides a sort of shared state and runtime between different layers. I think that is a good question as we will probably want to expose the context as part of the KableOptions.